rbac/internal/utils.py

```diff
@@ -95,6 +98,50 @@ def delete_bindings(bindings):
     return info
 
 
+def get_jwt_token(conn, grant_type, client_id, client_secret, scopes, url):
```

Comment:
this method can be in sso service class.

SSO service class has takes to get token - so maybe it can rename to JWTTokenProvider, so something like this. In that case,  all methods which are for to get token should be here. (calling RH sso,..)